jade/extensions/registry.py

```diff
@@ -51,6 +56,7 @@ class ExtensionClassType(enum.Enum):
 class Registry:
     """Manages extensions registered with JADE."""
     _REGISTRY_FILENAME = ".jade-registry.json"
+    FORMAT_VERSION = "v0.2.0"
```

Comment:
Two places have `FORMAT_VERSION = "v0.2.0"`, here and job_configuration.py, maybe place it at one place?

Reply:
They describe different things and may change independently.